src/config/links.js

```diff
@@ -91,7 +91,11 @@ export const links = {
   r_docs_tximport:
     'https://www.rdocumentation.org/packages/tximport/versions/1.0.3/topics/tximport',
   salmon: 'https://combine-lab.github.io/salmon/',
-  salmon_quant: 'https://combine-lab.github.io/salmon/'
+  salmon_quant: 'https://combine-lab.github.io/salmon/',
+  webhooks: {
+    // base urls for APIs
+    hubspot: 'https://api.hubapi.com/crm/v3/'
+  }
```

Comment:
This also doesn't belong in `config.links`. Should consistent with feedback from other PRs

Reply:
I matched the feedback from other PRs, pls see the commit at 3880d59ee752189c79ab237c3d8735dd850275b8 👍